cohortextractor/mssql_utils.py

```diff
@@ -93,6 +102,24 @@ def _ctds_connect(ctds, params):
     return ctds.connect(**params)
 
 
+def stats_msg_handler(msgstate, severity, srvname, procname, line, msgtext):
```

Comment:
I think this handler should probably have a big `try: ... except Exception: pass` around the body. We don't want any errors in stats parsing to stop the job from running.